test/distributed/fsdp/test_wrap.py

```diff
@@ -650,6 +701,116 @@ def test_auto_wrap_with_ignored_modules(self, wrap_method: WrapMethod):
         self.assertTrue(isinstance(model.module[2][0], nn.Linear))
         self.assertTrue(isinstance(model.module[2][1], nn.Linear))
 
+    @unittest.skipIf(torch.cuda.device_count() < 2, "Requires at least 2 GPUs")
```

Comment:
This doesn't have to be fixed this PR, but it does seem strange to me that we cannot run the "check valid param" logic without having more than two GPUs.

Reply:
oh, I guess this is the end to end test